gradle/publication.gradle

```diff
@@ -1,20 +1,31 @@
 apply plugin: 'maven-publish'
 apply plugin: 'signing'
 
-android {
-    publishing {
-        singleVariant('release') {
-            withSourcesJar()
-            withJavadocJar()
+if (pluginManager.hasPlugin("com.android.library")) {
+    android {
+        publishing {
+            singleVariant('release') {
+                withSourcesJar()
+                withJavadocJar()
+            }
         }
     }
+} else {
+    java {
```

Comment:
I've verified this works by using `publishToMavenLocal` and inspecting the output